docs/tutorials/computers101.md

```diff
@@ -0,0 +1,248 @@
+# How to Computers
+!!! Overview 
+    This page will provide guidance and key points for navigating and operating with the Unix Terminal, Git, Python, and SSH. 
+    Concepts, useful command charts, and demonstration of function are below. See <a href="Software-Carpentry.org">Software Carpentry</a>
+    for in depth instruction. 
+
+<a name="toc"></a> **Jump to:**
+- [How to Terminal](#terminal)
+- [How to `git` and GitHub](#git)
+- [How to `python`](#python)
+- [How to SSH](#ssh)

```

Comment:
I added this as we discussed, @charlottemeredith

Reply:
I like this feature and think it should indeed be incorporated.